From f2f26d51c862dbfda515e4b46b85c05b5c87e24e Mon Sep 17 00:00:00 2001
From: Jochen Breuer <jbreuer@suse.de>
Date: Wed, 6 Sep 2017 10:16:51 +0200
Subject: [PATCH] Catching error when PIDfile cannot be deleted

Usually the PIDfile is locate in /run. If Salt is not started with root
permissions, it is not able to delete the PIDfile in /run. It should
be safe to just log and ignore this error, since Salt overwrites the
PIDfile on the next start.
---
 tests/unit/utils/parsers_test.py | 1 +
 1 file changed, 1 insertion(+)

diff --git a/tests/unit/utils/parsers_test.py b/tests/unit/utils/parsers_test.py
index 9f1ad26219..a42abf0f4c 100644
--- a/tests/unit/utils/parsers_test.py
+++ b/tests/unit/utils/parsers_test.py
@@ -22,6 +22,7 @@ import salt.utils.parsers
 import salt.log.setup as log
 import salt.config
 import salt.syspaths
+from salt.utils.parsers import DaemonMixIn
 
 ensure_in_syspath('../../')
 
-- 
2.17.1


